examples/basic_mysql_connection/index.js

```diff
@@ -4,17 +4,27 @@ var mysql = require('mysql');
 var config = JSON.parse(fs.readFileSync("config-secret.json"))
 
 var connection = mysql.createConnection({
-	host: config.host,
+	 host: config.host,
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
+const sqlStatments=['SELECT COUNT(*) as "Total No of Todos" FROM todos',
+					'SELECT * FROM todos WHERE Done',
+					'SELECT * FROM todos WHERE NOT Done',	
+					'SELECT * FROM todos GROUP BY Id DESC',
```

Comment:
Hi Odai,
The changes look perfect for me, I would assume that you meant to use ORDER BY here instead of GROUP BY, right?